ArdusatSDK.cpp

```diff
@@ -1,28 +1,40 @@
 /**
  * @file   ArdusatSDK.cpp
- * @Author Ben Peters (ben@ardusat.com)
+ * @author Ben Peters (ben@ardusat.com)
+ * @author Sam Olds (sam@ardusat.com)
  * @date   December 3, 2014
  * @brief  Implements ArdusatSDK generic sensor reading and configuration for Space Kit Sensors.
+ *
+ * Provides a unifying wrapper of sensor specific functionality to provide a
+ * consistent interface to interact with each type of sensor.
  */
 
 #include <stdio.h>
 #include <string.h>
 #include "ArdusatSDK.h"
 
-bool ARDUSAT_SHIELD = false;
+bool MANUAL_CONFIG = false;
+bool ARDUSAT_SPACEBOARD = false;
```

Comment:
So, somewhat confusingly, Arduino defines the `boolean` data type in addition to the C++ primitive. It's kinda a toss-up as to which to use, but I think we should be consistent across the library, and it's a bit more "Arduino-like" to use `boolean` (even though I don't like it).